lighthouse-core/report/report-generator.js

```diff
@@ -323,7 +330,7 @@ class ReportGenerator {
     return template({
       url: results.url,
       lighthouseVersion: results.lighthouseVersion,
-      generatedTime: this._formatTime(results.generatedTime),
+      generatedTime: results.generatedTime,
```

Comment:
pass generated time as time instead of string to ease time comparison in perf-x-left-nav